int32 robot_id
float64 range
float64 bearing

